src/docs/config.toml

```diff
@@ -2,10 +2,11 @@
 # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
 # The Universal Permissive License (UPL), Version 1.0
 #
-baseURL = "https://oracle-quickstart.github.io/oci-cloudnative/"
+baseURL = ""
```

Comment:
can be omitted?

Reply:
yes, maybe, trying later. :)

Maybe need to remove the Site.BaseURL requests from other places

Reply:
There is another approach that I've used for this - which is to configure the action with the `--baseURL` flag for `hugo` command.

Reply:
@mvandervliet I thought about that, but why? any benefits? I could not find any.

Reply:
nah not really. Thought it might be useful for the forks to match on other branches... then just incorporate logic into the action on whether or not to _actually_ deploy